repos/system_upgrade/el7toel8/actors/yumupgradability/actor.py

```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.models import Inhibitor
+from leapp.tags import IPUWorkflowTag, ExperimentalTag, InitRamStartPhaseTag
+from leapp.libraries.stdlib import call
+
+
+class YumUpgradability(Actor):
+    """
+    An actor that takes care of yum upgradability.
```

Comment:
Please be more specific what it odes. Basically what you describe down there in secure_yum_upgradability would be more suitable here